protos/flyteidl/admin/launch_plan.proto

```diff
@@ -121,10 +121,9 @@ message LaunchPlanSpec {
     int32 max_parallelism = 18;
 
     // Allows for the interruptible flag of a workflow to be overwritten for a single execution.
-    // Omitting this field uses the workflow's value as a default.
+    // Omitting this field uses the launch plan's value as a default.
```

Comment:
oh wait i think this change doesn't make sense, let's keep the reference to workflow :)